aim/tests/unit/agent/aid_universes/test_converter.py

```diff
@@ -3923,6 +3944,30 @@ class TestAimToAciConverterVmmInjHost(TestAimToAciConverterBase,
     ]
 
 
+class TestAimToAciConverterNetflow(TestAimToAciConverterBase,
+                                   base.TestAimDBBase):
+    sample_input = [
+        base.TestAimDBBase._get_example_aim_netflow(),
+        base.TestAimDBBase._get_example_aim_netflow(name='netflow')]
```

Comment:
We should use this one to overwrite those default parameters in the _get_example_aim_netflow().